src/worker.ts

```diff
@@ -0,0 +1,99 @@
+import ReadableStream = NodeJS.ReadableStream;
+import EventEmitter = NodeJS.EventEmitter;
```

Comment:
That's very strange... The Travis build passed which runs `tsc`... Maybe delete the `out` folder and run `tsc` again?